src/compile-middleware.ts

```diff
@@ -144,6 +144,7 @@ export function browserNeedsCompilation(userAgent: string): boolean {
   const supportsES2015 = (browser.name === 'Chrome' && majorVersion >= 49) ||
       (browser.name === 'Chromium' && majorVersion >= 49) ||
       (browser.name === 'OPR' && majorVersion >= 36) ||
+      (browser.name === 'Vivaldi' && majorVersion >= 1) ||
```

Comment:
For those not familiar, is there a link which describes which version gained ES6 support and what the UA string is?